opencti-platform/opencti-front/src/private/components/events/incidents/IncidentCreation.tsx

```diff
@@ -284,11 +290,14 @@ const IncidentCreation = ({
   paginationOptions: IncidentsLinesPaginationQuery$variables;
 }) => {
   const { t_i18n } = useFormatter();
+  const { isFeatureEnable } = useHelper();
   const updater = (store: RecordSourceSelectorProxy) => insertNode(store, 'Pagination_incidents', paginationOptions, 'incidentAdd');
+  const FABReplaced = isFeatureEnable('FAB_REPLACEMENT');
   return (
     <Drawer
       title={t_i18n('Create an incident')}
-      variant={DrawerVariant.create}
+      variant={FABReplaced ? undefined : DrawerVariant.create}
+      controlledDial={FABReplaced ? CreateEntityControlledDial('entity_Incident') : undefined}
```

Comment:
I know this has been already merged, but I'm really not fond of this factory CreateEntityControlledDial.

IMO a more "react" solution would be to have a component `EntityControlledDial` that has the entityType in props:

```ts
import React from 'react';
import { Button } from '@mui/material';
import { useFormatter } from 'src/components/i18n';

interface EntityControlledDialProps {
  entityType: string
  onOpen: () => unknown
}

const EntityControlledDial: React.FC<EntityControlledDialProps> = ({ onOpen, entityType }) => {
  const { t_i18n } = useFormatter();
  const buttonValue = `${t_i18n('Create')} ${t_i18n(entityType)}`;
  return (
    <Button
      onClick={onOpen}
      color='primary'
      size='small'
      variant='contained'
      aria-label={buttonValue}
      title={buttonValue}
      sx={{
        marginLeft: '10px',
        padding: '7px 10px',
      }}
    >
      <div style={{ display: 'flex' }}>
        {buttonValue}
      </div>
    </Button>
  );
};

export default EntityControlledDial;
```

And then define specific ControlledDials where we need to. For instance here in IncidentCreation.tsx

```ts
//...
const IncidentControlledDial = (props: Omit<EntityControlledDialProps, 'entityType'>) => (
  <EntityControlledDial entityType={'entity_Incident'} {...props} />
);
//...
  <Drawer
      title={t_i18n('Create an incident')}
      variant={FABReplaced ? undefined : DrawerVariant.create}
      controlledDial={FABReplaced ? IncidentControlledDial : undefined}
    >

```